collector/settings.mdx

```diff
@@ -174,6 +174,11 @@ helper functions are expected to be defined, and the one we connect to for serve
       <td>[none]</td>
       <td>Alternative to above, using actual contents of the key</td>
     </tr>
+    <tr>
+      <td>db_data_directory (<code>DB_DATA_DIRECTORY</code>)</td>
+      <td>[autodetected]</td>
```

Comment:
It appears we use "auto-detected" without the brackets for other settings (see e.g. aws_db_instance_id), might be worth standardizing on one style.